osflow/build_arty.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I think this file is particular for your local setup. Instead of adding it, I would document it in the README (https://github.com/stnolting/neorv32-setups/tree/main/osflow#prerequisites). The important fact here is that the environment needs to be setup, anyhow.